napari/_qt/qt_main_window.py

```diff
@@ -336,6 +350,19 @@ def _add_help_menu(self):
         )
         self.help_menu.addAction(about_key_bindings)
 
+    def _toggle_axes(self, action):
+        text = action.text()
+        if text == 'None':
+            self.qt_viewer.viewer.axes_visible = False
+        elif text == 'Colored':
+            self.qt_viewer.viewer.axes_visible = True
+            self.qt_viewer.viewer.axes_style = 'Colored'
+        elif text == 'Dashed':
```

Comment:
Is dashed only in black and white?

* If possible, I'd prefer to keep the colours (unless that looks super horrible when you convert it to grayscale, like you might for a paper figure). Random thought: is it worth adding a grayscale keyword argument to our `nbscreenshot` method? If set to true, it could just run the scikit-image `rgb2grey` function on the output screenshot.
* I think one of the dashed lines (maybe the shorter dashes) should be made a little bit bold, if possible. As you rotate an object in 3D, the perspective on the dashed lines changes so it's sometimes hard to tell which dashed line is which.

Reply:
I could make all of the options independent - so you could have 
 - visible or not visible
- colored or not colored
- dashed or not dashed

You'd be able to set them individually in the dropdown menu and get a checkmark as appropriate. Default would probably be `not visible`, `colored`, and `not dashed`. I kinda like this idea. Thoughts @GenevieveBuckley @Czaki ?

> Random thought: is it worth adding a grayscale keyword argument to our nbscreenshot method? If set to true, it could just run the scikit-image rgb2grey function on the output screenshot.

I like that idea, but we don't depend on skimage so would have to do the grayscale conversion ourselves, which isn't too hard.

> I think one of the dashed lines (maybe the shorter dashes) should be made a little bit bold, if possible. As you rotate an object in 3D, the perspective on the dashed lines changes so it's sometimes hard to tell which dashed line is which.

Yeah, i noticed that too, it's not idea, good idea though - I can try that!!

Reply:
ok, i've given it a go with making `visible`, `colored`, and `dashed` all independent bools, I think it's much nicer now, but curious what others think!!